### DISCUSSION TITLE
Do some x-rays come from outside the ED?

### DISCUSSION THREAD

**Discussion by adviksh** [MIT-LCP/mimic-code#1719] [MIMIC-IV-CXR] [answered]
Hello all,

I'm trying to figure out when chest x-rays happen during a patient's journey through BIDMC. The MIMIC-CXR documentation states:
> We queried the BIDMC EHR for chest x-ray studies made in the emergency department between 2011 - 2016, and extracted the set of patient identifiers associated with these studies. We subsequently extracted all chest x-ray studies for this set of patients between 2011 - 2016.

**1. Does this mean that some chest x-rays in MIMIC-IV-CXR are conducted outside the emergency department?** 
For example, suppose a patient comes to the ED in 2012, has a first x-ray, is admitted to the hospital, has a second x-ray, and is discharged. They return to BIDMC in 2014 (but not through the ED) and have a third x-ray. Should all three x-rays appear in MIMIC-CXR?

**2. If non-ED x-rays do appear, how does BIDMC decide which radiologist reads them?** 
My understanding of the ED is that radiologists work in shifts, and whoever's on staff reads an x-ray when it comes up. Is this the case for non-ED x-rays as well, or are they assigned to radiologists some other way?

Thank you in advance for any guidance on either question!

**Comment by alistairewj** [comment, marked as answer]
> Hello all,
> 
> I'm trying to figure out when chest x-rays happen during a patient's journey through BIDMC. The MIMIC-CXR documentation states:
> > We queried the BIDMC EHR for chest x-ray studies made in the emergency department between 2011 - 2016, and extracted the set of patient identifiers associated with these studies. We subsequently extracted all chest x-ray studies for this set of patients between 2011 - 2016.
> 
> **1. Does this mean that some chest x-rays in MIMIC-IV-CXR are conducted outside the emergency department?** 
> For example, suppose a patient comes to the ED in 2012, has a first x-ray, is admitted to the hospital, has a second x-ray, and is discharged. They return to BIDMC in 2014 (but not through the ED) and have a third x-ray. Should all three x-rays appear in MIMIC-CXR?

Yes, correct, AFAIK. You could (and should) verify this by looking for someone with a non ED admit one year after their ED admit.

> **2. If non-ED x-rays do appear, how does BIDMC decide which radiologist reads them?** 
> My understanding of the ED is that radiologists work in shifts, and whoever's on staff reads an x-ray when it comes up. Is this the case for non-ED x-rays as well, or are they assigned to radiologists some other way?

Yes, there's no practical difference in how radiology works based on ED vs non-ED, even though the ED is technically a clinic. The hospital PACS is shared.

**Reply by adviksh** [reply]
Thank you, this answers my questions!

For future readers, you can verify the first answer by looking at admission 23258342 (hadm_id) for subject 10014610 (subject_id). The patient has ED visits before and after this hospital visit, but they aren't linked to the hospital visit. There are several x-ray studies associated with the hospital visit, like study 54258223 (study_id) for example.
